admin/verdaccio.yaml

```diff
@@ -13,11 +13,32 @@
 
 storage: ../storage
 
+## verdaccio does not allow to publish packages when the client is offline, to avoid
+## errors on publish it is disabled
+## https://verdaccio.org/docs/configuration#offline-publish
+publish:
+  allow_offline: false
+
 # A list of other known repositories we can talk to
 uplinks:
   npmjs:
     url: https://registry.npmjs.org/
 packages:
+  # this package is hosted outside the monorepo
+  # https://github.com/slorber/react-loadable
+  # thus requires to be fetched from npmjs
+  '@docusaurus/react-loadable':
```

Comment:
Is there a way to dynamically figure out which packages are in the local monorepo? In the future we may have more scoped (forked) packages not in the main repo.

Reply:
Well, since is committed in the repo is less flexible, but I could try adding the `proxy`, so any package that match with the namespace is lookup outside so the block above is not need it, it has disadvantages like for every package published will check on npmjs (here is when external network might break your build again).

```
  '@docusaurus/*':
     access: $all
     publish: $all
     proxy: npmjs
```

Let me know what do you prefer and I can update the config. 

Anyhow, I'm working in a different ways, like https://github.com/verdaccio/verdaccio/blob/master/packages/verdaccio/test/unit/proxy-multilpe-registry.spec.ts#L20 programatically generate the yaml files or stuff like that for the next major.

Reply:
Ah, good to know. For now, I think this hardcoded list is fine—we just anticipate another 2~3 forked dependencies + a few to be moved under this scope (ref #6246)